internal/bootstrap/bootstrap.go

```diff
@@ -246,8 +248,10 @@ func writeDotEnv(keys []api.ApiKeyResponse, config pgconn.Config, fsys afero.Fs)
 		case POSTGRES_DATABASE:
 			initial[k] = config.Database
 		case NEXT_PUBLIC_SUPABASE_ANON_KEY:
+		case EXPO_PUBLIC_SUPABASE_ANON_KEY:
```

Comment:
```suggestion
			fallthrough
		case EXPO_PUBLIC_SUPABASE_ANON_KEY:
```

Crazy thing about Golang switch case is that it breaks by default. So we need an explicit fall through here